rack-mini-profiler.gemspec

```diff
@@ -41,7 +41,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop-discourse'
   s.add_development_dependency 'listen'
   s.add_development_dependency 'webpacker'
-  s.add_development_dependency 'rails', '~> 6.0'
+  s.add_development_dependency 'rails', '>= 7.1'
```

Comment:
This resolves the `uninitialized constant ActiveSupport::LoggerThreadSafeLevel::Logger (NameError)` error

https://github.com/rails/rails/issues/54263